contracts/test/TestJBveBanny.t.sol

```diff
@@ -174,10 +252,214 @@ contract JBveBannyTests is TestBaseWorkflow {
     jbOperatorStore().setOperator(
       JBOperatorData(address(_jbveBanny), _projectId, _permissionIndexes)
     );
-    _jbveBanny.lock(_projectOwner, 10 ether, 864000, _projectOwner, false, false);
+    _jbveBanny.lock(_projectOwner, 10 ether, 604800, _projectOwner, false, false);
+    assert(_jbveBanny.tokenVotingPowerAt(1, block.number) > 0);
+    (int128 _amount, , uint256 _duration, bool _useJbToken, bool _allowPublicExtension) = _jbveBanny
+      .locked(1);
+    assertEq(_amount, 10 ether);
+    assertEq(_duration, 604800);
+    assert(!_useJbToken);
+    assert(!_allowPublicExtension);
     assertEq(_jbveBanny.ownerOf(1), _projectOwner);
-    (uint256 amount, uint256 duration, , ,) = _jbveBanny.getSpecs(1);
+    (uint256 amount, uint256 duration, , , ) = _jbveBanny.getSpecs(1);
     assertEq(amount, 10 ether);
-    assertEq(duration, 864000);
+    assertEq(duration, 604800);
+  }
+
+  function testLockVotingPowerIncreasesIfLockedLonger() public {
+    mintIJBTokens();
+
+    _jbveBanny.lock(_projectOwner, 5 ether, 604800, _projectOwner, true, false);
+    assert(_jbveBanny.tokenVotingPowerAt(1, block.number) > 0);
+
+    _jbveBanny.lock(_projectOwner, 5 ether, 2419200, _projectOwner, true, false);
+    assert(_jbveBanny.tokenVotingPowerAt(2, block.number) > 0);
+
+    // Since lock-2 is 4x as long as lock-1, it should have x4 the voting power
+    // (might be slightly more or less due to rounding to nearest week)
+    assertGt(
```

Comment:
'slightly more or less' but you test only for greather than? iirc, forge-std has a 'approx equals to', allowing small variations from rounding (or use the same method to compute the theoretical result, to get the same rounding approximation)

Reply:
missed this will add